server.js

```diff
@@ -21,7 +14,52 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json({responseMessage: "NES-games library at /games"});
+});
+
+app.get("/games", (req, res) => {
+  const { title, publisher, developer } = req.query;
+  let games = nesGames;
+
+  if (publisher) {
+    games = games.filter((singleGame) => { return singleGame.publisher.toLowerCase() === publisher.toLowerCase() });
+  }
+  if (developer) {
+    games = games.filter((singleGame) => { return singleGame.developer.toLowerCase() === developer.toLowerCase() });
+  }
+  if (title) {
+    games = games.filter((singleGame) => { return singleGame.title.toLowerCase() === title.toLowerCase() });
+  }
+
+  res.status(200).json({
+    success: true,
+    message: "OK",
+    response: {
+      nesGames: games
+    }
+  });
+
+});
+
+app.get("/games/:id", (req, res) => {
```

Comment:
When it comes to the ID, I disovered that it might be good to use like /games/id/:id, because if you (or someone else working with your code) wants to add a new endpoint outside the collection you created, the browser will get confused and misinterpret eaither the ID or the other new endpoint, and can't read it. Super confusing comment, but I hope you understand what I mean!